### PR TITLE
Create nodistribute.com ruleset

### PR DESCRIPTION
**Pull request by zero77**

### CODE REVIEW

**Comment by J0WI** [comment]
Please use tabs to match out code style.
Rule lgtm.

**Comment by J0WI** [comment]
@zero77 are you still working on this?

**Comment by zero77** [comment]
No its finished.

**Comment by J0WI** [comment]
But you should use tabs instead of spaces.

**Comment by zero77** [comment]
i used tabs this time

**Comment by J0WI** [comment]
Nope, they are still spaces.

**Comment by zero77** [comment]
The spaces should be gone now.

File: src/chrome/content/rules/nodistribute.com.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<ruleset name="NoDistribute"
```
Comment (J0WI) [inline src/chrome/content/rules/nodistribute.com.xml]: Missing closing bracket here.

File: src/chrome/content/rules/nodistribute.com.xml [hunk 2]
```diff
@@ -1,5 +1,5 @@
-<ruleset name="NoDistribute" />
-	<target host="nodistribute.com" />
-	<target host="www.nodistribute.com" />
-	<rule from="^http:" to="https:" />
-</ruleset>
++<ruleset name="NoDistribute">
++	<target host="nodistribute.com" />
++	<target host="www.nodistribute.com" />
++ 	<rule from="^http:" to="https:" />
```
Comment (J0WI) [inline src/chrome/content/rules/nodistribute.com.xml]: Looks like you accidentally copy-pasted some `+` and space.

**Comment by J0WI** [comment]
All tests passed.
Thanks for contributing!

**Pull request merged**